source/standards/source-code/use-github.html.md.erb

```diff
@@ -70,18 +76,26 @@ Please double-check that the digest and the commented version are consistent eac
 
 Third-party actions should only be used if:
 
-- The provider is verified by GitHub (for example, [aws-actions](https://github.com/aws-actions))
+- The provider is verified by GitHub (for example, [aws-actions])
 - The action is complex enough that you cannot write your own local action
 - You have fully reviewed the code in the version of the third-party action you will be using
 - You have pinned the specific version in your workflow and in the repository settings, using a Git commit SHA
 - You have included the semver version in a comment next to the SHA, helping humans understand the version and automated tools report on what is out of date (for example dependabot)
 - The third-party action is actively maintained, well-documented and tested ([follow the guidance on third party dependencies](/standards/tracking-dependencies.html)).
 
-You can enforce this in the settings for Actions by choosing ['Allow select actions'](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#allowing-specific-actions-to-run) and then 'Allow actions created by GitHub' and 'Allow Marketplace actions by verified creators' as required.
+You can enforce this in the settings for Actions by choosing ['Allow select actions'][github-managing-github-actions-settings-for-a-repository]
+and then 'Allow actions created by GitHub' and 'Allow Marketplace actions by verified creators' as required.
 
 Note that for public repositories, the output of workflow runs is visible to everyone. Do not use workflows if this output could be considered sensitive.
 
-### Access GitHub support
+## How to retire applications
+
+If an application is no longer used in production, you should [archive its repository][github-archive-a-repo].
```

Comment:
There might be more then one repo
```suggestion
If an application is no longer used in production, you should [archive its repositories][github-archive-a-repo].
```

Reply:
other the rest of the paragraph is about a singular application/repo so I don't know if this is worth changing